src/group_theory/p_group.lean

```diff
@@ -118,6 +116,7 @@ lemma card_orbit (a : α) [fintype (orbit G a)] :
 begin
   let ϕ := orbit_equiv_quotient_stabilizer G a,
   haveI := fintype.of_equiv (orbit G a) ϕ,
+  haveI := (stabilizer G a).finite_index_of_finite_quotient,
```

Comment:
[is_p_group.index](https://leanprover-community.github.io/mathlib_docs/group_theory/p_group.html#is_p_group.index) should probably also switch to `finite_index`?

Reply:
This lemma is `is_p_group.index`, and it is being switched to `finite_index` (line 98).